### PR TITLE
fix: use Prettier config for formatting if available

### LINKED ISSUE #217: Make CLI auto-detect and read prettier.config.js
So config doesn't need to be duplicated with `--style.KEY=VALUE`

### PR DESCRIPTION
**Pull request by allout58**
Important note: because of this PR, all snapshots had to be
updated, as the tests will attempt to use the project's
Prettier config. This mostly just removed the semicolons.

closes #217

### CODE REVIEW

File: test/resources/prettier/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"bracketSpacing": true,
+	"printWidth": 80,
+  "endOfLine": "crlf",
+  "semi": true,
```
Comment (bcherny) [inline test/resources/prettier/.prettierrc]: Nit: Indentation

**Comment by jamesarosen** [comment]
Can I help get this merged?

**Comment by split** [comment]
Any plans getting this merged at some point? 🙏

**Comment by bcherny** [comment]
Closing out stale PRs. Feel free to rebase and re-open if you like.

**Pull request closed without merging**